`zig build` failed on Windows

```
zig-x86_64-windows-0.14.1\lib\std\posix.zig:1996:9: error: std.posix.getenv is unavailable for Windows because environment strings are in WTF-16 format. See std.process.getEnvVarOwned for a cross-platform API or std.process.getenvW for a Windows-specific API.
        @compileError("std.posix.getenv is unavailable for Windows because environment strings are in WTF-16 format. See std.process.getEnvVarOwned for a cross-platform API or std.process.getenvW for a Windows-specific API.");
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
referenced by:
    build: jedisct1\minisign\build.zig:45:29
    runBuild__anon_27712: zig-x86_64-windows-0.14.1\lib\std\Build.zig:2428:44
    5 reference(s) hidden; use '-freference-trace=7' to see all references
```

--- a/build.zig
+++ b/build.zig
@@ -42,14 +42,19 @@ pub fn build(b: *std.Build) !void {
         minisign.linkLibrary(libzodium);
     } else {
         var override_pkgconfig = false;
-        if (std.posix.getenv("LIBSODIUM_INCLUDE_PATH")) |path| {
+        var arena = std.heap.ArenaAllocator.init(std.heap.page_allocator);
+        defer arena.deinit();
+        const allocator = arena.allocator();
+        if (std.process.getEnvVarOwned(allocator, "LIBSODIUM_INCLUDE_PATH")) |path| {
             minisign.addSystemIncludePath(.{ .cwd_relative = path });
+            allocator.free(path);
             override_pkgconfig = true;
-        }
-        if (std.posix.getenv("LIBSODIUM_LIB_PATH")) |path| {
+        } else |_| {}
+        if (std.process.getEnvVarOwned(allocator, "LIBSODIUM_LIB_PATH")) |path| {
             minisign.addLibraryPath(.{ .cwd_relative = path });
+            allocator.free(path);
             override_pkgconfig = true;
-        }
+        } else |_| {}
 
         for ([_][]const u8{ "/opt/homebrew/include", "/home/linuxbrew/.linuxbrew/include", "/usr/local/include" }) |path| {
             std.fs.accessAbsolute(path, .{}) catch continue;

